### PR TITLE
Update how-atp-safe-links-works.md

### PR DESCRIPTION
**Pull request by ruiwensu**
Added info for office on the web and error case

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/security/office-365-security/how-atp-safe-links-works.md [hunk 1]
```diff
@@ -44,11 +44,11 @@ At a high level, here's how [ATP Safe Links](atp-safe-links.md) protection works
     
 ## How ATP Safe Links works with URLs in Office documents
 
-At a high level, here's how [ATP Safe Links](atp-safe-links.md) protection works for URLs in Office 365 ProPlus applications (current versions of Word, Excel, and PowerPoint on Windows or Mac, Office apps on iOS or Android devices, Visio on Windows, OneNote in a browser, and Office in a browser):
+At a high level, here's how [ATP Safe Links](atp-safe-links.md) protection works for URLs in Office 365 ProPlus or Business Premium applications (current versions of Word, Excel, and PowerPoint on Windows, Mac, or in a browser, Office apps on iOS or Android devices, Visio on Windows, OneNote in a browser):
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/how-atp-safe-links-works.md:47]:
```suggestion
At a high level, here's how [ATP Safe Links](atp-safe-links.md) protection works for URLs in Office 365 ProPlus or Business Premium applications (current versions of Word, Excel, and PowerPoint on Windows, Mac, or in a browser, Office apps on iOS or Android devices, Visio on Windows, or OneNote in a browser):
```

File: microsoft-365/security/office-365-security/how-atp-safe-links-works.md [hunk 2]
```diff
@@ -61,4 +61,6 @@ At a high level, here's how [ATP Safe Links](atp-safe-links.md) protection works
       - If the URL goes to a downloadable file and the [ATP Safe Links policies](set-up-atp-safe-links-policies.md) are configured to scan such downloads, the downloadable file is checked. 
     
       - If the URL is considered safe, the user is taken to the website.
+      
+      - If the URL check fails, Safe Links' protection will not trigger. On the desktop clients, the user will be warned before proceeding through to the site.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/how-atp-safe-links-works.md:65]:
```suggestion
      - If the URL check fails, Safe Links protection will not trigger. On the desktop clients, the user will be warned before proceeding through to the site.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy Review

**Pull request merged**